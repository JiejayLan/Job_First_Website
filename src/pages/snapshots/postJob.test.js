import React from 'react';
import ReactShallowRenderer from 'react-test-renderer/shallow';
import PostJob from "../postJob"

describe("postJob snapshot test", () => {
    test('postJob render should match snapshot', () => {

    });
});