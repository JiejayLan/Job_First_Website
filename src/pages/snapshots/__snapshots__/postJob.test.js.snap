// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`postJob snapshot test postJob render should match snapshot 1`] = `
<div
  align="center"
>
  <br />
  <h1>
    Post a New Job
  </h1>
  <Form
    className="main-form"
    colon={true}
    hideRequiredMark={false}
    layout="horizontal"
    onSubmit={[Function]}
  >
    <FormItem
      hasFeedback={false}
    >
      <Input
        disabled={false}
        placeholder="Enter Employer Name"
        prefix={
          <Icon
            type="user"
          />
        }
        suffix={
          <Tooltip
            arrowPointAtCenter={false}
            autoAdjustOverflow={true}
            mouseEnterDelay={0.1}
            mouseLeaveDelay={0.1}
            placement="top"
            title="Enter the name of the employer."
            transitionName="zoom-big-fast"
          >
            <Icon
              type="info-circle"
            />
          </Tooltip>
        }
        type="text"
      />
    </FormItem>
    <FormItem
      hasFeedback={false}
    >
      <PickerWrapper
        locale={Object {}}
        onChange={[Function]}
        onOk={[Function]}
        onOpenChange={[Function]}
        placeholder="Date Posted On"
        popupStyle={Object {}}
        transitionName="slide-up"
      />
    </FormItem>
    <FormItem
      hasFeedback={false}
    >
      <Select
        choiceTransitionName="zoom"
        placeholder="Job Type"
        showSearch={false}
        transitionName="slide-up"
      >
        <Option
          value="Full Time"
        >
          Full Time
        </Option>
        <Option
          value="Part Time"
        >
          Part Time
        </Option>
        <Option
          value="Internship"
        >
          Internship
        </Option>
        <Option
          value="Temporary"
        >
          Temporary
        </Option>
      </Select>
    </FormItem>
    <FormItem
      hasFeedback={false}
    >
      <TextArea
        autosize={
          Object {
            "maxRows": 6,
            "minRows": 2,
          }
        }
        placeholder="Enter Job Description"
      />
    </FormItem>
    <FormItem
      hasFeedback={false}
    >
      <Input
        disabled={false}
        placeholder="(###) ###-####"
        prefix={
          <Icon
            type="contacts"
          />
        }
        suffix={
          <Tooltip
            arrowPointAtCenter={false}
            autoAdjustOverflow={true}
            mouseEnterDelay={0.1}
            mouseLeaveDelay={0.1}
            placement="top"
            title="Enter the contact number of the employer."
            transitionName="zoom-big-fast"
          >
            <Icon
              type="info-circle"
            />
          </Tooltip>
        }
        type="text"
      />
    </FormItem>
    <FormItem
      hasFeedback={false}
    >
      <Button
        block={false}
        ghost={false}
        htmlType="submit"
        loading={false}
        type="primary"
      >
        Submit Job
      </Button>
    </FormItem>
  </Form>
</div>
`;
